### PR TITLE
[EUWE] Adds supported_consoles attribute and related changes

### PR DESCRIPTION
**Pull request by AllenBW**
https://bugzilla.redhat.com/show_bug.cgi?id=1505546
euwe pr for https://github.com/ManageIQ/manageiq-ui-service/pull/1280

WIP contingent on https://github.com/ManageIQ/manageiq/pull/16591

### CODE REVIEW

**Comment by simaishi** [comment]
@AllenBW https://github.com/ManageIQ/manageiq/pull/16591 is merged now

**Comment by simaishi** [comment]
@dclarizio @chalettu Not sure who can review/approve this PR...  this is for a blocker.

**Comment by dclarizio** [comment]
@AllenBW is this still WIP or ready for review/merge?

**Comment by AllenBW** [comment]
Now that  ManageIQ/manageiq#16591 is in this is g2g, @himdel at your earliest! 🙇

File: client/app/states/services/details/details.html [hunk 1]
```diff
@@ -234,10 +234,10 @@ <h2 translate>Virtual Machines ({{ ::vm.vms.length }})</h2>
               </span>
             </div>
             <div class="col-lg-1 col-md-1 hidden-sm hidden-xs">
-              <button class="btn btn-default open-console-button" ng-if="item['supports_console?'] && item['supports_vnc_console?'] && item.power_state == 'on'" tooltip="{{'HTML5 console'|translate}}" tooltip-placement="bottom">
+              <button class="btn btn-default open-console-button" ng-if="!item.supported_consoles.vnc.visible && !item.supported_consoles.vnc.enabled" tooltip="{{'HTML5 console'|translate}}" tooltip-placement="bottom">
```
Comment (himdel) [inline client/app/states/services/details/details.html]: @AllenBW looks like this is now missing that `&& item.power_state == 'on'` ?

(Assuming we still don't want to run consoles on powered-off VM's..)

Comment (AllenBW) [reply client/app/states/services/details/details.html]: @jntullo correct me if i'm wrong... 🤔

Comment (jntullo) [reply client/app/states/services/details/details.html]: @AllenBW you're correct. `enabled` ensures that it is powered on

Comment (himdel) [reply client/app/states/services/details/details.html]: OK, then the only remaining problem ... the condition is negative again :D ;)

Comment (himdel) [reply client/app/states/services/details/details.html]: I assume `ng-if="item.supported_consoles.vnc.visible && item.supported_consoles.vnc.enabled"`

**Pull request merged**